module/world-explorer-layer.mjs

```diff
@@ -184,6 +184,15 @@ export class WorldExplorerLayer extends foundry.canvas.layers.InteractionLayer {
         }
     }
 
+    activateEditingControls(toolName) {
```

Comment:
Can you rename this to onChangeTool? I think that's what's more accurate to what's going on here.